lib/active_model/serializer/reflection.rb

```diff
@@ -62,11 +62,12 @@ def build_association(subject, parent_serializer_options)
 
       private
 
-      def serializer_options(parent_serializer_options, reflection_options)
+      def serializer_options(parent_serializer_options, reflection_options, subject)
         serializer = reflection_options.fetch(:serializer, nil)
```

Comment:
Because, as any parent, it is its own subject, but its children's parent. The `serializer_options` is the hash that we build for passing down to the associated (i.e. child) serializer.